core/server/api_container/server/startosis_engine/kurtosis_instruction/exec/exec.go

```diff
@@ -17,11 +18,21 @@ import (
 	"go.starlark.net/starlark"
 )
 
+var defaultAcceptableCodes = []int{
+	0, // EXIT_SUCCESS
+}
+
 const (
 	ExecBuiltinName = "exec"
 
-	RecipeArgName      = "recipe"
-	ServiceNameArgName = "service_name"
+	RecipeArgName          = "recipe"
+	ServiceNameArgName     = "service_name"
+	AcceptableCodesArgName = "acceptable_codes"
+	SkipCodeCheckArgName   = "skip_code_check"
```

Comment:
code/codes ?

Reply:
lets add docs for these

Reply:
Added docs for both `exec` and `request`